doc/sphinx-guides/source/admin/integrations.rst

```diff
@@ -185,6 +185,16 @@ Avgidea Data Search
 
 Researchers can use a Google Sheets add-on to search for Dataverse installation's CSV data and then import that data into a sheet. See `Avgidea Data Search <https://www.avgidea.io/avgidea-data-platform.html>`_ for details.
 
+JupyterHub
+++++++++++
+
+The Dataverse-to-JupyterHub Data Transfer Connector streamlines data transfer between Dataverse repositories and the cloud-based platform JupyterHub, enhancing collaborative research.
```

Comment:
We should add a link, right? To https://forgemia.inra.fr/dipso/eosc-pillar/dataverse-jupyterhub-connector ?

Reply:
The link "https://forgemia.inra.fr/dipso/eosc-pillar/dataverse-jupyterhub-connector" was added.